src/SamplesApp/UITests.Shared/UITests.Shared.projitems

```diff
@@ -41,10 +41,6 @@
       <SubType>Designer</SubType>
       <Generator>MSBuild:Compile</Generator>
     </Page>
-    <Page Include="$(MSBuildThisFileDirectory)Windows_Devices\BarometerTests.xaml">
```

Comment:
This is an error... will be removed in another commit.

Was removed locally because not working on UWP.